Podfile.lock

```diff
@@ -1,59 +1,47 @@
 PODS:
-  - Alamofire (4.2.0)
-  - Appirater (2.0.5)
+  - Alamofire (4.6.0)
+  - Appirater (2.3.1)
   - Aspects (0.0.1)
-  - AsyncSwift (2.0.1)
+  - AsyncSwift (2.0.4)
   - CallbackURLKit (1.0.0):
     - CallbackURLKit/Core (= 1.0.0)
   - CallbackURLKit/Core (1.0.0)
-  - Cartography (1.0.1)
+  - Cartography (3.0.1)
```

Comment:
Are you sure `Cartogrphy 3.0.1` works with Potatso?
Did you try cleaning up the project before building? Thanks

Reply:
Yes, I had cleaning up before. no errors.